Framework/Backend/services/kafka.js

```diff
@@ -0,0 +1,127 @@
+/**
+ * @license
+ * Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+ * See http://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+ * All rights not expressly granted are reserved.
+ *
+ * This software is distributed under the terms of the GNU General Public
+ * License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+ *
+ * In applying this license CERN does not waive the privileges and immunities
+ * granted to it by virtue of its status as an Intergovernmental Organization
+ * or submit itself to any jurisdiction.
+*/
+
+const { Kafka, logLevel } = require('kafkajs')
+const WebSocketMessage = require('../websocket/message.js');
+const log = new (require('./../log/Log.js'))(`${process.env.npm_config_log_label ?? 'web-ui'}/kafka`);
+
+/**
+ * Gateway for all Kafka notification service
+ */
+class KafkaConnector {
+  /**
+   * Prepares connector
+   * @param {object} config Config with list of Kafka brokers
+   */
+  constructor(config) {
+    if (!config) {
+      log.warn('Missing configuration');
+      return;
+    }
+    if (!config.brokers || config.brokers.length < 1) {
+      throw new Error(`Kafka broker list was not provided`);
+    }
+    this.kafka = new Kafka({
+      clientId: 'webui',
+      brokers: config.brokers,
+      retry: {retries: 3},
+      logLevel: logLevel.NOTHING
+    });
+
+    this.admin = this.kafka.admin();
+    this.consumer = null;
+    this.webSocket = null;
+    log.info('Kafka connector configured');
+  }
+
+  /**
+   * Check if Kafka was correctly configured
+   * @return {boolean}
+   */
+  isConfigured() {
+    return (this.kafka !== undefined && this.admin !== undefined);
+  }
+
+  /**
+   * Provides healthstatus of Kafka cluster
+   */
+  health() {
+    return Promise.resolve()
```

Comment:
Could this and rest of the methods be written using `async/await` format?
Example from the `kafkajs` module: https://www.npmjs.com/package/kafkajs#usage

Reply:
yep definitelly